algotaurus.py

```diff
@@ -423,15 +425,16 @@ def __init__(self, size=15, lines=30):
         self.tkMessageBox = tkMessageBox
 
         # Initial parameters
-        self.size = size
         self.lines = lines
-        self.x = 27
-        self.y = 27
+        self.code = code
+        self.maze_rows = maze_rows
+        self.maze_columns = maze_columns
+        self.size = min(450 / maze_columns, 550 / maze_rows, 25)
```

Comment:
Replace x and y with maze_rows and maze_columns and put it in the class parameters.
Remove size from the class parameters and make it dependent from rows and columns number